### ISSUE TITLE
Web dashboard: Title, users and values are truncated

### ISSUE THREAD

**Issue by AdrianPotter** [ISISComputingGroup/IBEX#2316]
Reported by Helen Playford on Polaris.

On the web dashboard, the title and users are being truncated:

![image](https://cloud.githubusercontent.com/assets/10273443/25893246/f8502906-356e-11e7-8cf6-36529b98929b.png)
(ALF)

![image](https://cloud.githubusercontent.com/assets/10273443/25893323/3b8569d4-356f-11e7-9b62-b721a3a0a1c5.png)
(Polaris)

It appears that there may be some encoding issue. This has been observed both on Alf and Polaris. The current title and user on Polaris is:

![image](https://cloud.githubusercontent.com/assets/10273443/25893277/13d541ac-356f-11e7-8c37-199b5bae9833.png)

There's no obvious reason why this shouldn't be able to be displayed. There are no special characters in either the title or users.

Acceptance Criteria:

1. Blocks can be viewed
1. Title and user are not truncated
1. Errors are displayed for blocks in error (and only blocks in error)
1. Title and User name values are marked as unavailable unless the show title box on the DAE tab is ticked

**Comment by DominicOram** [comment]
The underlying cause of this is that the archive engine webpage (that we are using to get the title/username) truncates the data. We will have to get the data from somewhere else, SQL? EPICS? then serve it via the JSON the blockserver is sending to the web dashboard.

**Comment by DominicOram** [comment]
Note that this was originally reported in https://github.com/ISISComputingGroup/IBEX/issues/1627. I will close that one as this is better defined.

**Comment by Tom-Willemsen** [comment]
POLARIS have also reported that this truncation happened to some of their numbers, causing some confusion.

Apparently a `furnace_vac` block displayed as `0.000...`on the web dashboard when the real value was `0.0005`. (Number of zeroes is illustrative only)

Inform Ron/Helen on POLARIS when this ticket is done as they seemed quite keen for this to be fixed.

**Comment by DominicOram** [comment]
@Tom-Willemsen I think that should be a separate issue as I think they might have quite different causes/solutions. The truncation of values has already been noted in https://github.com/ISISComputingGroup/IBEX/issues/2069

**Comment by Tom-Willemsen** [comment]
OK, will copy my comment over to that ticket.

**Comment by kjwoodsISIS** [comment]
Plan for resolving this problem will be determined by outcome of #2463

**Comment by Tom-Willemsen** [comment]
Requires #2477

**Comment by John-Holt-Tessella** [comment]
Pull request https://github.com/ISISComputingGroup/JSON_bourne/pull/20

**Comment by John-Holt-Tessella** [comment]
This has utf8 issues see #2728  for ticket to fix this.

**Comment by kjwoodsISIS** [comment]
Are we saying this ticket is now dependent on #2728?  (i.e. it impeded until #2728 is completed?)

**Comment by John-Holt-Tessella** [comment]
No, this ticket can be merged when done and I am hoping will provide the functionality for most use cases. However #2728 is a separate important fix that needs to be done at some point.
